projects/common/lib/components/date/date.component.ts

```diff
@@ -250,7 +250,13 @@ You must use either [restrictDate] or the [dateRange*] inputs.
       const month = this.getNumericValue(this._month);
       const day = this.getNumericValue(this._day);
       // console.log('CREATING DATE', { year, month, day });
-      this.date = new Date(year, month, day);
+
+      // Date function appears to use setYear() so any year 0-99 results in yar 1900 to 1999
+      // Set each field individually, use setFullYear() instead of setYear()
+      this.date  = new Date();
+      this.date .setFullYear( year );
```

Comment:
Remove spaces after `this.date` in this and all following lines.  It's valid, but looks weird.

```suggestion
      this.date.setFullYear( year );
```